<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>영상 실험 GUI (PyQt6)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_root" stretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="gridContainer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>700</width>
          <height>814</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_previews">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <property name="spacing">
          <number>8</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="preview_3">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="preview_5">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="preview_8">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>8</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="preview_2">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="preview_9">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>9</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="preview_4">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="preview_7">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>7</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="preview_6">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGraphicsView" name="preview_1"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="controlPanel">
       <layout class="QVBoxLayout" name="verticalLayout_ctrl">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_buttons">
          <item>
           <widget class="QPushButton" name="btnRun">
            <property name="text">
             <string>실행</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExit">
            <property name="text">
             <string>종료</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QLabel" name="labelPath">
          <property name="text">
           <string>비디오: (미선택)</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupParams">
          <property name="title">
           <string>파라미터</string>
          </property>
          <layout class="QFormLayout" name="formLayout_params">
           <item row="0" column="0">
            <widget class="QLabel" name="labelS1">
             <property name="text">
              <string>슬라이더 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="slider1">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelS2">
             <property name="text">
              <string>슬라이더 2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSlider" name="slider2">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelS3">
             <property name="text">
              <string>슬라이더 3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSlider" name="slider3">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelS4">
             <property name="text">
              <string>슬라이더 4</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSlider" name="slider4">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelS5">
             <property name="text">
              <string>슬라이더 5</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSlider" name="slider5">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
